qemu/cpu/code_generation.py

```diff
@@ -528,12 +530,70 @@ def fill_env_get_cpu_body(cputype, function):
         )
     )
 
-def fill_gdb_rw_register_body(cputype, function, comment):
+gdb_get_reg_bitsizes = (8, 16, 32, 64, 128)
+ld_size_infixes = {16: "uw", 32: "l", 64: "q"}
+
+def fill_gdb_rw_register_body(cputype, function, is_write = False):
     cpu = Pointer(Type[cputype.struct_instance_name])("cpu")
     env = Pointer(Type[cputype.struct_name])("env")
 
+    n = function.args[2]
+
+    reg_number = 0
+    cases = []
+    for reg in cputype.registers:
+        bank_size = reg.bank_size
+        raw_bitsize = reg.raw_bitsize
+        env_reg = OpSDeref(env, reg.name)
+
+        if bank_size:
+            env_reg = OpIndex(
+                env_reg,
+                OpSub(n, reg_number) if reg_number else n
+            )
+            case_val = CaseRange(reg_number, reg_number + bank_size - 1)
+            reg_number += bank_size
+        else:
+            case_val = reg_number
+            reg_number += 1
+

```

Comment:
When `raw_bitsize` does not match any existing helper, the correctness of this boilerplate is questionable. At least, a warning for developer should be printed.

Reply:
Looks like, there is also a mismatch between maximum read/write accessible sizes. If we don't support 128 writes, we should at least warn a user.

Reply:
`raw_bitsize` is always at most 64 bits, as the `CPURegister` constructor checks for this, so there is always a suitable helper. The 128 read size was added only because such a helper is present in the Qemu code, but never used in boilerplate.

Reply:
But, the constructor still accepts values like 42. So, just let a user to know, that he (she) a stupid one.

Add:
- a warning `print`;
- a comment with an explanation for developer.